source/_docs/https-load-balancer.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Supported Methods for Configuring an HTTPS Load Balancer
+description: Explore different solutions for enabling HTTPS on all environment URLs (including Multidev) associated with your organization.
```

Comment:
The driving use-case for most customers here is that they need HTTPS across all sites (including personal sites) more than they are worried about multidev environments.